scripts/data_ingestion/ingest_current_season.py

```
@@ -0,0 +1,220 @@
+    if result.stderr:
+        print(result.stderr, file=sys.stderr)
+    
+    success = result.returncode == 0
+    print(f"Status: {'✅ SUCCESS' if success else '❌ FAILED'}")
+    return success
+
+
+def fetch_schedule(season: int | None = None, days_back: int | None = None) -> bool:
+    """Fetch MLB schedule for current season or recent days."""
+    if season:
+        cmd = ["python3", "scripts/fetch_mlb_schedule.py", f"--season", str(season)]
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)